tests/functional/test_oauth.py

```diff
@@ -120,12 +107,29 @@ def test_oauth_routes_support_cors_preflight(self, app, method, path):
         app.options(
             path,
             headers={
-                "Origin": str("https://third-party-client.herokuapp.com"),
+                "Origin": "https://third-party-client.herokuapp.com",
```

Comment:
Wat? Is this some kind of python 2 thing?

Reply:
Yes. It is obsolete. See 9edc6fbed062ce64efdf84051293c0a7f6b0f212 and other commits listed by `git log -S native_str`.